lib/client.js

```diff
@@ -704,19 +704,19 @@ class Client extends EventEmitter {
    *   }
    * );
    */
-  confirmedCOVNotification(address, monitoredObject, subscribeId, initiatingDeviceId, lifetime, values, options, next) {
+  confirmedCOVNotification(receiver, monitoredObject, subscribeId, initiatingDeviceId, lifetime, values, options, next) {
```

Comment:
Why the rename here?
- Using `receiver` seems a bit too generic to me (how to address a receiver?).
- The parameter for addressing devices should be in-line trough-out all client functions and not only change for a single one.
- The change should be reflected in the JSDoc comment above.

Reply:
This was simply for consistency.  I found `address` confusing when working with BBMD code (is it your address, the sender's address, or the recipient's address?) so I used 'receiver' as this word was already used by other functions (e.g. [`readPropertyResponse`](https://github.com/fh1ch/node-bacstack/blob/master/lib/client.js#L1128)).  You are welcome to use a different word but the important part is that they are all the same (better not to mix `address` and `receiver` as exists in the current `master` branch.)

I am not sure what you mean by the parameter being in-line, as this change makes `receiver` behave the same for all functions, and it can be a string as before.